install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -241,10 +241,6 @@ spec:
         {{- end }}
         {{- end }}
         securityContext:
-          capabilities:
-            add:
-            - NET_ADMIN
```

Comment:
should these capabilities be cleanup from from other k8s manifest files (e.g. test/k8sT/manifests/*) as well ?

Reply:
Good call! I did a quick grep and found some more redundant capabilities. I removed them all.